CHANGELOG.md

```diff
@@ -51,6 +51,9 @@
   [David Jennes](https://github.com/djbe)
   [#595](https://github.com/SwiftGen/SwiftGen/issue/595)
   [#600](https://github.com/SwiftGen/SwiftGen/pull/600)
+* Strings: template option to force add file name `enum` structure even if only single file is provided (unsupported for swift3).
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/82ca1c71fe58e4bc7acb0a1e9d25d2077fbaf41e/CHANGELOG.md#L54"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>